docs/sources/upgrade-guide/upgrade-v8.0/index.md

```diff
@@ -0,0 +1,64 @@
+---
+aliases:
+
```

Comment:
Remove empty line
```suggestion
```

Same for other upgrade guides.

Reply:
👍